### PR TITLE
test(quantic): E2E tests for ResultTemplate

### PR DESCRIPTION
**Pull request by aelfaleh**
image

### CODE REVIEW

File: packages/quantic/cypress/integration/result-template/result-template.cypress.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      Expect.displaySlotByName('actions', true);
+      Expect.displaySlotByName('date', true);
+      Expect.displaySlotByName('visual', true);
+      Expect.displaySlotByName('title', true);
+      Expect.displaySlotByName('metadata', true);
+      Expect.displaySlotByName('emphasized', true);
+      Expect.displaySlotByName('excerpt', true);
+      Expect.displaySlotByName('bottom-metadata', true);
+
+      Actions.appendChildren('div', {
```
Comment (nathanlb) [inline packages/quantic/cypress/integration/result-template/result-template.cypress.ts:28]: Are you planning on appending stuff to every section? If so then maybe using a map of the slot names and a forEach loop would be a nice way of doing it.

**Review by lvu285 — APPROVED** [review]

File: packages/quantic/cypress/integration/result-template/result-template-expectations.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+} from './result-template-selectors';
+
+function resultTemplateExpectations(selector: ResultTemplateSelector) {
+  return {
+    displaySlotByName: (name: string, display: boolean) => {
+      selector
+        .slotByName(name)
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(
+          `${should(display)} display the element by name equal "${name}"`
```
Comment (lbergeron) [inline packages/quantic/cypress/integration/result-template/result-template-expectations.ts:14]:
```suggestion
          `${should(display)} display the element named "${name}"`
```

**Review by lbergeron — COMMENTED** [review]
It is looking good so far. What is missing at this point?

File: packages/quantic/cypress/integration/result-template/result-template.cypress.ts (hunk 1 above)
Comment (aelfaleh) [reply packages/quantic/cypress/integration/result-template/result-template.cypress.ts:28]: I use a enum for that 😆

**Review by nathanlb — APPROVED** [review]
pretty awesome!

**Pull request merged**